x11/lightdm-kde-greeter/Makefile

```diff
@@ -0,0 +1,31 @@
+PORTNAME=	lightdm-kde-greeter
+PORTVERSION=	6.0.3
+CATEGORIES=	x11 kde
+
+MAINTAINER=	kde@FreeBSD.org
+COMMENT=	KDE login screen using the LightDM framework
+WWW=		https://invent.kde.org/plasma/lightdm-kde-greeter
+
+LIB_DEPENDS=	liblightdm-gobject-1.so:x11/lightdm \
+		libKF6NetworkManagerQt.so:net-mgmt/networkmanager-qt
+RUN_DEPENDS=	at-spi2-core>0:accessibility/at-spi2-core
+
+LICENSE=	GPLv3
+LICENSE_FILE=	${WRKSRC}/COPYING
+
+USES=		cmake compiler:c++11-lang gettext gnome kde:6 pkgconfig qt:6 \
+		tar:xz xorg
+USE_KDE=	auth config configwidgets i18n iconthemes kcmutils package \
+		plasma-workspace \
+		ecm:build
+USE_GNOME=	gtk20
+USE_QT=		base
+USE_XORG=	x11
+
+KDE_INVENT=	31d29530ad834c1bfb70ed43f9395a549fbace56 plasma
+
+CMAKE_ARGS=	-DGREETER_IMAGES_DIR="${PREFIX}/lib/${PORTNAME}/images" \
```

Comment:
This one looks strange. Installing images into `/lib/`? I also don't see anything like that in the pkg-plist, so maybe this isn't needed at all?

Reply:
It's needed for the build to not complain but yeah that is odd. That might explain why I didn't see any background images when loading lightdm-kde-greeter.